package.json

```diff
@@ -28,7 +28,7 @@
     "watch": "watch 'npm test && npm run lint' ./lib ./test"
```

Comment:
Should me mentioned in CNR (as we have recently done in similar PRs at PEP and Perseo-FE)